modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+[id="cluster-limitations-local-zone_{context}"]
+= Cluster limitations in AWS Local Zones
+
+Some limitations exist when you attempt to deploy a cluster with a default installation configuration in Amazon Web Services (AWS) Local Zones.
+
+[IMPORTANT]
+====
+The following list details limitations when deploying a cluster in AWS Local Zones:
+
+- The Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in a region is `1300`. This causes the cluster-wide network MTU to change according to the network plugin that is used on the deployment.
+- Network resources such as Network Load Balancer (NLB), Classic Load Balancer, and Nat Gateways are not supported in AWS Local Zones.
+- The `gp3` AWS Elastic Block Storage (EBS), which is the default storage class for node volumes and AWS deployments, is not globally available on Local Zone locations. By default, the nodes running in Local Zones are deployed with the `gp2` EBS volume. The `gp2-csi` storage class must be set when creating workloads on Local Zone nodes.
```

Comment:
@mtulio can we include a link to the docs where we explain how to create a Storage Class on day-2?

Reply:
@makentenza for sure. Pointing to our documentation we have this section: https://docs.openshift.com/container-platform/4.12/storage/understanding-persistent-storage.html#pvc-storage-class_understanding-persistent-storage

I strongly believe we can provide one example/day-2 docs using Local Zones and gp2-csi StorageClass, as our existing documentation is generic for AWS[1]. We are doing similar work exemplifying the deployment in Local Zones[2], it can be appended or based on it. WDYT?

[1] https://docs.openshift.com/container-platform/4.12/storage/generic-ephemeral-vols.html
[2] https://github.com/openshift/openshift-docs/pull/57063

Reply:
I can move this update to https://github.com/openshift/openshift-docs/pull/57063

Reply:
Add the link to the Storage Class doc